src/components/DpTreeList/DpTreeListNode.vue

```diff
@@ -94,6 +95,12 @@
             v-bind="scope" />
         </template>
       </dp-tree-list-node>
+      <li
+        v-if="isBranch && children.length <= 0 && hasDraggableChildren"
```

Comment:
why `&& hasDraggableChildren`?  i understand that this element serves as a drop zone if there are no leaves yet. However in that case, hasDraggableChildren would be false, i guess?

Reply:
hasDraggableChildren indicates only if it could have draggable children. Even if there are no children yet, this is true. It "only" checks if draggable is true and the leaves are draggable:
```
 this.isBranch && (this.options.dragLeaves || this.options.dragBranches) && this.draggable
 ```